data-structures/filecoin/messages.md

```diff
@@ -0,0 +1,422 @@
+# Filecoin Messages Data Structures
```

Comment:
is it possible to somehow have the mappings from method Number to struct type for these? They are an implicit union keyed on MethodNum within an actor.